account_invoice_force_removal/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def unlink(self):
+        """Allow to remove moves related to invoices."""
+        for item in self:
+            if item.type in ["out_invoice", "out_refund", "in_invoice", "in_refund"]:
+                if item.state == "cancel" and item.name != "/":
+                    item.write({"name": "/"})
```

Comment:
Is it ' /'  the default value that allows to unlink or is it `False`?

Reply:
Is '/', https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L1742

Reply:
OK, but seeing this, we can use the context `force_delete` and not doing any write.